test/fixtures.hpp

```diff
@@ -106,6 +106,25 @@ class TwoDFixture : public testing::Test {
   }
 };
 
+class TwoDSimpleNormalizationFixture : public testing::Test {
+protected:
+    RegularGridInterpolator test_rgi;
+    GriddedData test_gridded_data;
+    std::vector<std::vector<double>> values;
+    std::vector<double> target;
+
+    TwoDSimpleNormalizationFixture() {
+        std::vector<std::vector<double>> grid = {{2.0, 7.0}, {1.0, 2.0, 3.0}};
+        //          1.0   2.0   3.0
+        values = {{ 2.0,  2.0,  6.0,    // 2.0
```

Comment:
This should be 4, I've fixed it. Great catch. We could calculate values as x*y, I am happy to do so. I formatted the data as shown because this is the format of both the unit tests in EnergyPlus and also the format of the other test fixtures inside of btwxt. Your call.

Reply:
Data is now generated through a function :)